include/WCSimPMTObject.hh

```diff
@@ -16,9 +16,9 @@ public:
   virtual G4double GetExposeHeight()=0;
   virtual G4double GetRadius()=0;
   virtual G4float* Getqpe()=0;
-  virtual G4float* GetQE()=0;
-  virtual G4float* GetQEWavelength()=0;
-  virtual G4float  GetmaxQE()=0;
+  virtual G4double* GetQE()=0;
```

Comment:
Otherwise you'll get a "narrowing conversion" error in C++11. You'll need double math inside the array.